src/parser-typescript.js

```diff
@@ -35,7 +35,10 @@ function tryParseTypeScript(text, jsx) {
     tokens: true,
     comment: true,
     useJSXTextNode: true,
-    ecmaFeatures: { jsx }
+    ecmaFeatures: { jsx },
+    // Override logger function with noop,
+    // to avoid unsupported version errors being logged
+    loggerFn: Function.prototype
```

Comment:
Oh, I always use it as it does not create an unnecessary function and is the only true noop in JS AFAIK... Happy to change it though